trunk/source/Library-Automata/src/de/uni_freiburg/informatik/ultimate/automata/petrinet/operations/PrefixProduct.java

```diff
@@ -73,7 +72,7 @@
 	private final Map<PLACE, PLACE> mOldPlace2newPlace = new HashMap<>();

```

Comment:
You removed a similar map in `PrefixProduct` -- would it make sense to do the same here?

Reply:
I guess you mean `Difference`. If it behaves similarly, I can also remove it here.

Reply:
Yes, I meant `Difference` -- sorry